### ISSUE TITLE
SheetOverflowException exception message is bogus

### ISSUE THREAD

**Issue by pchote** [OpenRA/OpenRA#12718]
```
Exception of type `OpenRA.Graphics.SheetOverflowException`: Cross-sheet depth sprite reference: {0}.{1}: {2}
```

Repro case: start the TS mod with `Graphics.SheetSize=1024`.

**Comment by yaaaaa** [comment]
hmmm... when i put  isodepth.shp from TS bits folder into RA2 bits folder, i have same exception.
Can i use this file in RA2 mod to add NAHPAD structure? Or how can i add NAHPAD without use this file?

**Comment by GraionDilach** [comment]
This should be fixed with #15194.

**Comment by pchote** [comment]
Indeed it was.
